src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/UserService.kt

```diff
@@ -58,10 +60,15 @@ class UserService(
     telemetryClient.trackEvent("ExternalUserDisabled", mapOf("username" to user.name, "admin" to authenticationFacade.getUsername()), null)
   }
 
+  suspend fun getAllUsersLastName(): List<UserLastNameDto> =
+    userRepository.findAllBySource().map { UserLastNameDto(it.getUserName(), it.lastName!!) }.toList()
```

Comment:
can this return null or does it return an empty list if none are found?

Reply:
shouldn't otherwise there are big problems - no users found means whole service is broken